Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -83,6 +83,32 @@ def bump_formula_pr
       checked_for_duplicates = true
     end
 
+    linux_core = OS.linux? && formula.tap.core_tap?
```

Comment:
Might be worth pulling all this into another function?

Reply:
Yes, I thought about that too. Will surely do!

Reply:
Done initially, but don't know if it's properly to the end.